### PR TITLE
#1678 - add conversion

### LINKED ISSUE #1678: Convert m-sum of m-array to m-sum array
... by one MinkowskiSumArray.

e.g the result in the second line could be automatically simplified to a `MinkowskiSumArray{Float64,Zonotope{Float64}}`:

```julia
julia> z = rand(Zonotope);

julia> typeof(z ⊕ MinkowskiSumArray([z, z]))
MinkowskiSum{Float64,Zonotope{Float64},MinkowskiSumArray{Float64,Zonotope{Float64}}}
```

### PR DESCRIPTION
**Pull request by mforets**
Closes #1678.

### CODE REVIEW

File: src/convert.jl [hunk 1]
```diff
@@ -893,3 +893,23 @@ function convert(::Type{VPolygon},
     @assert dim(X) == 2
     return VPolygon(vertices_list(X))
 end
+
+"""
+    convert(::Type{MinkowskiSumArray},
+            X::MinkowskiSum{N, ST, MinkowskiSumArray{N, ST}}) where {N, ST}
+
+Converts the minkowski sum of a minkowski sum array to a minkowski sum array.
```
Comment (schillic) [inline src/convert.jl]: We always used a capital M.

File: src/convert.jl [hunk 2]
```diff
@@ -893,3 +893,23 @@ function convert(::Type{VPolygon},
...
+
+### Input
+
+- `MinkowskiSumArray`  -- type used for dispatch
+- `X`                  -- minkowski sum of a minkowski sum array
```
Comment (mforets) [inline src/convert.jl]:
```suggestion
- `X`                  -- Minkowski sum of a Minkowski sum array
```

File: src/convert.jl (hunk 1 above)
Comment (mforets) [inline src/convert.jl]:
```suggestion
Converts the Minkowski sum of a Minkowski sum array to a Minkowski sum array.
```

File: src/convert.jl [hunk 3]
```diff
@@ -893,3 +893,23 @@ function convert(::Type{VPolygon},
...
+Converts the Minkowski sum of a Minkowski sum array to a Minkowski sum array.
+
+### Input
+
+- `MinkowskiSumArray`  -- type used for dispatch
+- `X`                  -- Minkowski sum of a Minkowski sum array
+
+### Output
+
+A minkowski sum array.
```
Comment (mforets) [inline src/convert.jl]:
```suggestion
A Minkowski sum array.
```

**Pull request merged**